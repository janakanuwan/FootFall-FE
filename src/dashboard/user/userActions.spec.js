import {logoutUser} from "./userActions";

describe('userActions', () => {
  it('should create an action for user logout', () => {
    expect(logoutUser(1)).toMatchSnapshot();
  });
});


