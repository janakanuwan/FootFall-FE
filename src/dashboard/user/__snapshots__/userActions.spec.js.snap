// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`userActions should create an action for user logout 1`] = `
Object {
  "payload": Object {
    "user": 1,
  },
  "type": "LOGOUT_USER",
}
`;
